SPECS/expat.spec

```diff
@@ -0,0 +1,257 @@
+Summary: An XML parser library
+Name: expat
+Version: 2.1.0
```

Comment:
A 2.1.0 el package as a base to 2.5.0 seems old.  The XS import indeed seems not too far from the el9 2.5.0 imported in `upstream-8.2`